projects/js-packages/social-previews/src/bluesky-preview/post/actions/styles.scss

```
@@ -0,0 +1,19 @@
+.bluesky-preview__post-actions {
+	margin-top: 0.5rem;
+	display: flex;
+	justify-content: space-between;
+	align-items: center;
+
+	/* stylelint-disable-next-line scss/selector-no-redundant-nesting-selector */
+	& > div {
```

Can this not be done this way?
```suggestion
	> div {
```